### ISSUE TITLE
Failure in test_query_txt_bytes1

### ISSUE THREAD

**Issue by gvanem** [saghul/pycares#66]
The `test/test.py` fails on ` test_query_txt_bytes1` since the domain `like.com.ca` does not longer exists:
```
FAIL: test_query_txt_bytes1 (__main__.DNSTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "F:\MingW32\src\inet\DNS\c-ares\PyCares\tests\tests.py", line 206, in test_query_txt_bytes1
    self.assertEqual(self.errorno, None)
AssertionError: 12 != None
```

12 == `ARES_ETIMEOUT`.
But changing it to `like.com`, all seems okay:
```diff
--- a/tests/tests.py 2017-11-16 21:09:56
+++ b/tests/tests.py 2018-06-07 16:23:10
@@ -201,7 +201,7 @@
         self.result, self.errorno = None, None
         def cb(result, errorno):
             self.result, self.errorno = result, errorno
-        self.channel.query('like.com.sa', pycares.QUERY_TYPE_TXT, cb)
+        self.channel.query('like.com', pycares.QUERY_TYPE_TXT, cb)
         self.wait()
         self.assertEqual(self.errorno, None)
```

**Comment by 0mp** [comment]
I can confirm that the test fails. I'm testing on FreeBSD 11.2.

**Comment by saghul** [comment]
Care to send a PR? I'm a bit busy these days, sorry..

**Comment by 0mp** [comment]
I think it was fixed in 03e9320
